### ISSUE TITLE
What does the loss function's value finally converge to?

### ISSUE THREAD

**Issue by zshyang** [gouthamvgk/SuperGlue_training#15]
Hi, 

I am training on my dataset. And I am using the same loss function. I saw the loss begin at 3.xxx and gradually drop to 1.9xxx. What is your experience with the scope of loss values?

Best regards
